remoteappmanager/docker/image.py

```diff
@@ -33,7 +33,9 @@ def from_docker_dict(cls, docker_dict):
         except (KeyError, IndexError):
             self.name = ''
 
-        labels = docker_dict.get("Labels")
+        labels = (docker_dict.get("Labels") or
+                  docker_dict.get("Config", {}).get("Labels"))
+
```

Comment:
If I remember correctly, we decided to use a consistent strategy for this, that is, not to pass the images() dict, but the inspect_image dict instead. We already encountered this problem with the containers IIRC. Please check if we are consistent.